src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import, print_function
+
+import logging
+
+from datetime import timedelta
+
+from django.core.cache import cache
+from django.utils import timezone
+
+from sentry.eventstore.models import Event
+from sentry.eventstore.processing import event_processing_store
+from sentry.models import Project, Release
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils import metrics
+from sentry.utils.committers import get_serialized_event_file_committers
+
+PREFERRED_GROUP_OWNERS = 2
+MIN_COMMIT_SCORE = 2
+OWNER_CACHE_LIFE = 3600  # seconds
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+GROUP_PROCESSING_DELAY = timedelta(
+    minutes=10
```

Comment:
Just recording conversational points from Slack here:
- The main reason was ensuring accuracy and striking a balance between re-calculating every event for "perfect" accuracy and say...never calculating. 
- Finding the best balance would likely involve investigation and the metrics will help with this.
- Using scores can be a good way to not replace more relevant owners, and to prevent any flip-flop owner issues in merged groups.